src/components/icons/Icon.tsx

```diff
@@ -379,7 +379,8 @@ export const IOIcons = {
   systemSettingsiOS: IconSystemSettingsiOS,
   systemToggleInstructions: IconSystemToggleInstructions,
   systemAppsAndroid: IconSystemAppsAndroid,
-  systemNotificationsInstructions: IconSystemNotificationsInstructions
+  systemNotificationsInstructions: IconSystemNotificationsInstructions,
+  doubleSarsFilled: IconStarFilled
```

Comment:
```suggestion
  doubleStarsFilled: IconDoubleStarsFilled
```
I don't know if you checked out the example app, but `IconStarFilled` is not the right component, according to the new component added in the `svg` folder.

Also consider moving the key up, because the last keys are used for other sets (as you can see from the `system` prefix).